core/src/create.jl

```diff
@@ -17,21 +17,66 @@ function LinearLevelConnection(db::DB, config::Config)::LinearLevelConnection
     return LinearLevelConnection(tbl.node_id, tbl.conductance)
 end
 
+"""
+For a `node_id` and a vector of such IDs, get the range of indices of the
```

Comment:
I modified it a bit. Since this only supports a specific usecase, I'm not trying to write the most generic code in general. I'm often a bit conservative in the type signature, e.g. using `Int` here. If we need to use this for other types we can remove this restriction and add tests for the other use cases.